addons/survey/models/survey_user.py

```diff
@@ -276,6 +281,29 @@ def _get_line_comment_values(self, question, comment):
             'value_char_box': comment,
         }
 
+    def _get_answer_score_values(self, question, answer, answer_type):
+        # default and non-scored questions
```

Comment:
It's good to have inline comments, but we usually prefer docstring instead.
I would put a small description of this method's purpose in the docstring, saying that you compute the score for both choice and regular questions where it's applicable. 

(You can keep the inline comments though).

Reply:
Docstring added.